### PR TITLE
Fix part of #1874: fix span test for Recently Played Screen

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix part of #1874 
Span test is Home moved to new `RecentlyPlayedSpanTest`

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/test/java/org/oppia/android/app/testing/RecentlyPlayedSpanTest.kt [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+class RecentlyPlayedSpanTest {
+
+  @Inject
+  lateinit var context: Context
+
+  @Inject
+  lateinit var testCoroutineDispatchers: TestCoroutineDispatchers
+
+  @Inject
+  lateinit var profileTestHelper: ProfileTestHelper
```
Comment (rt4914) [inline app/src/test/java/org/oppia/android/app/testing/RecentlyPlayedSpanTest.kt]: This is not needed.

**Review by rt4914 — CHANGES_REQUESTED** [review]
PTAL @anandwana001

File: app/src/test/java/org/oppia/android/app/testing/RecentlyPlayedSpanTest.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/test/java/org/oppia/android/app/testing/RecentlyPlayedSpanTest.kt]: ah, my bad, sorry. Done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**